src/utils/getters/getDoubleDigitTime.js

```diff
@@ -0,0 +1,3 @@
+export function getDoubleDigitTime(time) {
```

Comment:
FYI, you could replace all functions with blocks, where you just immediately `return` something into one-liners with arrow functions.
For example: 
```
export const getDoubleDigitTime(time) => (time < 10 ? "0": "") + time;
```
Of course don't overuse this syntax sugar, readibility is key.